### PR TITLE
Allow to disable a popover menu button

### PR DESCRIPTION
**Pull request by MorrisJobke**

### CODE REVIEW

File: src/components/PopoverMenu/PopoverMenuItem.vue [hunk 1]
```diff
@@ -71,7 +71,7 @@
 
 		<!-- If item.action is set instead, a button will be used -->
 		<button v-else-if="item.action" class="menuitem focusable" :class="{active: item.active}"
-			@click.stop.prevent="item.action">
+			@click.stop.prevent="item.action" v-bind:disabled="item.disabled">
```
Comment (skjnldsv) [inline src/components/PopoverMenu/PopoverMenuItem.vue]: please use the shortand `:disabled` :)

Comment (MorrisJobke) [reply src/components/PopoverMenu/PopoverMenuItem.vue]: Mind to do the change? Feel free to merge this then as well and bring it in the server 👍

**Comment by skjnldsv** [comment]
> This popovermenu is deprecated in favor of an Action element now, any particular reason you came to edit this file? :)

@MorrisJobke ^

**Comment by MorrisJobke** [comment]
> This popovermenu is deprecated in favor of an Action element now, any particular reason you came to edit this file? :)

It is used in the update channel selector of the update notification app

**Comment by skjnldsv** [comment]
I guess we should aim for the new Actions now then, no?
@ChristophWurst ?

**Comment by MorrisJobke** [comment]
> It is used in the update channel selector of the update notification app

https://github.com/nextcloud/server/pull/15514

File: src/components/PopoverMenu/PopoverMenuItem.vue (hunk 1 above)
Comment (skjnldsv) [inline src/components/PopoverMenu/PopoverMenuItem.vue]:
```suggestion
			@click.stop.prevent="item.action" :disabled="item.disabled">
```

**Comment by skjnldsv** [comment]
Let's get this in then

**Pull request merged**